source/_static/css/custom.css

```
@@ -0,0 +1,4 @@
+div.highlight-shell-session + div.highlight-default pre {
```

Tested this with the following

````
```shell-session
foo bar
```

    baz
````

and it did not show a difference, because the CSS class in that case is `hightlight-none`.

```suggestion
div.highlight-shell-session + div.highlight-none pre {
```